src/callbacks.rs

```diff
@@ -117,8 +117,22 @@ pub extern fn view_request_move(view: WlcView, _dest: &Point) {
     }
 }
 
-pub extern fn view_request_resize(_view: WlcView, _edge: ResizeEdge, _location: &Point) {
-
+pub extern fn view_request_resize(view: WlcView, edge: ResizeEdge, point: &Point) {
+    if let Ok(mut tree) = try_lock_tree() {
+        match try_lock_action() {
+            Ok(guard) => {
+                if guard.is_some() {
+                    if let Some(id) = tree.lookup_view(view) {
+                        if let Err(err) = tree.resize_container(id, edge, *point) {
+                            error!("Context: Trying to resize a container for the user \
```

Comment:
Ha, I was trying out that new error reporting that I read in that blog post...I totally forgot about it, so I think I'll just remove it here. It's confusing if it's only in one place ;).

I wouldn't say our logs are a user interface, the point of this error was to help the debugger (eg us) understand what the user was doing without having to mentally recreate the scene.